signing_service/signing_service/signing_service.py

```diff
@@ -384,6 +410,28 @@ def _get_authentication_challenge_signature(self, authentication_challenge: byte
             authentication_challenge,
         )
 
+    @staticmethod
+    def _get_signing_service_daily_transaction_sum_so_far() -> int:
+        thresholds_directory = (Path.cwd()).joinpath('daily_tresholds')  # pylint: disable=no-member
```

Comment:
This path should be constant (or configurable with default?). What do you guys think @dybi @bartoszbetka

Reply:
I would do this as simple as it can get:
```python
directory_path = os.path.join('.', 'daily_tresholds')
daily_file = os.path.join(directory_path, datetime.datetime.now().strftime('%Y-%m-%d'))
```

Reply:
If I we `os.path` instead of `pathlib.Path` to do all that, we are actually overcomplicating that function. I will have to write things like:
```
try:
    os.makedirs(directory)
except OSError as e:
    ...
```
when creating directory and something similar when creating file.
`pathlib.Path` makes all of that for us.

Reply:
What I have meant is maybe we should make this path configurable in case if deployment would like to keep it elsewhere on the file system. @bartoszbetka Do you have some thoughts about this?

Reply:
I see two different solution here:
- This directory and files within are temporary.
This solution is good.
- This directory and files within are persistent.
In this case @rwrzesien is right. We want to path to this directory are configurable. This solution also requires create and mount additional disk. We also make sure that directory was creating after disk mounted, so after the first connection.

Reply:
By temporary, I mean for every deployment this directory will be removed.

Reply:
I think that first solution is fine, but I'll wait for @dybi clarification.